src/engraving/libmscore/property.cpp

```diff
@@ -405,6 +405,8 @@ static constexpr PropertyMetaData propertyList[] = {
     { Pid::CAPO_IGNORED_STRINGS,    true,  "ignoredStrings",        P_TYPE::INT_VEC,        DUMMY_QT_TR_NOOP("propertyName", "ignored strings") },
     { Pid::CAPO_GENERATE_TEXT,      true,  "generateText",          P_TYPE::BOOL,           DUMMY_QT_TR_NOOP("propertyName", "automatically generate text") },
 
+    { Pid::TIE_PLACEMENT,           true,  "tiePlacement",          P_TYPE::TIE_PLACEMENT,            DUMMY_QT_TR_NOOP("propertyName", "tie placement") },
```

Comment:
Nitpick: better reduce the space bevore that `DUMMY...` to make it align with the others